vignettes/Introduction.Rmd

```diff
@@ -50,7 +50,7 @@ Person <- R6Class("Person",
 )
 ```
 
-To instantiate an object of this class, use `$new()`:
+To instantiate an object of this class, use `$new()` method:
```

Comment:
I prefer this the old way, without "method", because it slightly reduces the chance of confusion about generator methods vs. instance methods.

Reply:
Makes sense. I've reverted this change.